.github/workflows/build_image.yml

```diff
@@ -0,0 +1,26 @@
+name: Build docker image
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+  
+jobs:
+
+    build-docker:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v2
+        - uses: docker/build-push-action@v1
+          with:
+            path: docker/validator
+            repository: ${{ secrets.DOCKERHUB_USERNAME }}/tf-hub
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
+            password: ${{ secrets.DOCKERHUB_PASSWORD }}
+            dockerfile: docker/validator/Dockerfile
```

Comment:
do we need this? it defaults to `{path}/Dockerfile` or on newer version  {context}/Dockerfile` so we can drop this line